stock_request/__manifest__.py

```diff
@@ -9,7 +9,7 @@
     "website": "https://github.com/OCA/stock-logistics-warehouse",
     "author": "ForgeFlow, Odoo Community Association (OCA)",
     "category": "Warehouse Management",
-    "depends": ["stock"],
+    "depends": ["stock", "mrp"],
```

Comment:
You need to make another module as people don't want to pull mrp app if they are using stock only.